radiomanager/radiomanager/__init__.py

```diff
@@ -34,14 +23,47 @@ class TESTING_CONFIG:
     SQLALCHEMY_DATABASE_URI = "sqlite://"
 
 
-def create_app(config_obj=PRODUCTION_CONFIG):
+def create_app(config_obj=TESTING_CONFIG):
     from .api import setup_routing
     app = Flask(__name__)
     app.config.from_object(config_obj)
     setup_routing(FlaskExtensions.api)
     with app.app_context():
         FlaskExtensions.api.init_app(app)
         FlaskExtensions.db.init_app(app)
+        FlaskExtensions.db.create_all()
+    @app.route('/', methods=['GET','POST'])
+    def homepage():
+
+        MUMBLE_SERVICE = "net.sourceforge.mumble.murmur"
+        bus = dbus.SystemBus()
+        server = bus.get_object(MUMBLE_SERVICE, "/1")
+        # playerList = server.getPlayers()
+        murmur = dbus.Interface(server, 'net.sourceforge.mumble.Murmur')
+        devices = murmur.getPlayers(dbus_interface='net.sourceforge.mumble.Murmur')
+        # devices = Device.query.all()
+        print(devices, file=sys.stderr)
+        outputList = []
+        for d in devices:
+            device = { 
+                'name': d[8],
+                'radioID': d[7],
+                'channel': d[6],
+                'onlineseconds': d[9]
+            }
+            outputList.append(device)
+
+        # for device in devices:
```

Comment:
This commented out block should probably be removed.

Reply:
removed